docs/auth.md

```diff
@@ -41,40 +41,47 @@ refers to `TaskRuns` and `PipelineRuns` as `Runs` for the sake of brevity.
 
 ## Overview
 
-Tekton supports authentication via the Kubernetes first-class `Secret` types listed below.
+## Overview
```

Comment:
This is duplicated heading 🙃

Reply:
Sorry for the typo. Will resolve this, as soon as possible